### ISSUE TITLE
hasEdge(source, target) for MultiDirectedGraph

### ISSUE THREAD

**Issue by nielinjie** [graphology/graphology#431]
hasEdge(source, target) for MultiDirectedGraph return false. Is this a bug or a feature?

```
import { DirectedGraph, MultiDirectedGraph } from "graphology"

describe.only("a graph",()=>{
    it("hash Edge for Directed",()=>{
        const g = new DirectedGraph()
        const na = g.addNode("A")
        const nb = g.addNode("B")
        const eid = g.addEdge("A","B",{})
        expect(g.hasEdge("A","B")).toEqual(true) //ok
        expect(g.hasEdge(na,nb)).toEqual(true) //ok
        expect(g.hasEdge(eid)).toEqual(true)
    })
    it('hash Edge for MultiDirected', () => {
      const g = new MultiDirectedGraph();
      const na = g.addNode('A');
      const nb = g.addNode('B');
      const eid = g.addEdge('A', 'B', {});
      expect(g.hasEdge('A', 'B')).toEqual(true); //fails
      expect(g.hasEdge(na, nb)).toEqual(true); //fails
      expect(g.hasEdge(eid)).toEqual(true);
    });
})
```

**Comment by Yomguithereal** [comment]
Hello @nielinjie you are right indeed. This bug was introduced by a recent multi edge adjacency refactoring as doubly linked lists. I just committed a fix. I must now fix another bug linked to degree computation without self loops in the multi case and then I will release a v0.25.1 of `graphology`.

**Comment by Yomguithereal** [comment]
@nielinjie the fix is now released as v0.25.1, along with a correction for degree without self loops methods.
